ergotree-interpreter/src/eval/sheader.rs

```diff
@@ -58,22 +61,31 @@ pub(crate) static HEIGHT_EVAL_FN: EvalFn = |_env, _ctx, obj, _args| {
 
 pub(crate) static MINER_PK_EVAL_FN: EvalFn = |_env, _ctx, obj, _args| {
     let header = obj.try_extract_into::<Header>()?;
-    Ok(header.miner_pk.into())
+    Ok(header.autolykos_solution.miner_pk.into())
 };
 
 pub(crate) static POW_ONETIME_PK_EVAL_FN: EvalFn = |_env, _ctx, obj, _args| {
     let header = obj.try_extract_into::<Header>()?;
-    Ok(header.pow_onetime_pk.into())
+    Ok(header
+        .autolykos_solution
+        .pow_onetime_pk
+        .unwrap_or_else(|| Box::new(EcPoint::default()))
```

Comment:
Just want to check with you that the following default value is fine. Recall that `pow_onetime_pk` doesn't need to be specified in a `Header` JSON representation if we're dealing with autolykos v2.

Reply:
Good call!